src/libmongoc/src/mongoc/mongoc-stream-tls-openssl.c

```diff
@@ -343,15 +343,15 @@ _mongoc_stream_tls_openssl_writev (mongoc_stream_t *stream,
                _mongoc_stream_tls_openssl_write (tls, to_write, to_write_len);
             if (bson_cmp_not_equal_su (child_ret, to_write_len)) {
                TRACE ("Got child_ret: %ld while to_write_len is: %ld",
-                      child_ret,
-                      to_write_len);
+                      (long)child_ret,
+                      (long)to_write_len);
             }
 
             if (child_ret < 0) {
                TRACE ("Returning what I had (%ld) as apposed to the error "
                       "(%ld, errno:%d)",
-                      ret,
-                      child_ret,
+                      (long)ret,
+                      (long)child_ret,
                       errno);
                RETURN (ret);
```

Comment:
Same suggestion here, rather than cast to long, use `%zu` format specifier for `size_t` types.